src/Microsoft.AspNetCore.Mvc.Core/RequireHttpsAttribute.cs

```diff
@@ -43,6 +47,13 @@ protected virtual void HandleNonHttpsRequest(AuthorizationFilterContext filterCo
                     request.Path.ToUriComponent(),
                     request.QueryString.ToUriComponent());
```

Comment:
Does `UriBuilder` work better with or without the puny-coding that `ToUriComponent()` adds?